### PR TITLE
Basic "by graph"s

### PR DESCRIPTION
**Pull request by vincerubinetti**
- add env vars
- readme with overview info
- add jsdoc comment formatting prettier plugin
- add import sorter script
- fill out index html with meta data
- add favicon, share thumb, etc
- add d3 for graphs, zustand for state, tippy for tooltips
- gzip csvs
- basic layout components Cols and Placeholder
- add tippy tooltip with data-tooltip attrs
- setup comlink web workers parsing of data
- add basic bar graphs for sample count by phylum and class
- tweak header viz

### CODE REVIEW

File: src/data/worker.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    /** get name from most specific part */
+    const name = _class || phylum || kingdom;
+
+    /** count number of non-zero rows in col */
+    let samples = 0;
+    for (let row = 1; row < data.length; row++)
+      if (data[row][col] !== "0") samples++;
+
+    /** number */
+    if (name !== "NA")
```
Comment (rabdill) [inline src/data/worker.ts:53]: Would it be quicker to check this before looping through all the rows? If it's "NA," we can just continue to the next column, right?

Comment (vincerubinetti) [reply src/data/worker.ts]: Indeed. I've moved it above the totaling step, but still after the `name` step. I figure we want to only skip it if the most specific level is exactly equal to `NA`?

Comment (rabdill) [reply src/data/worker.ts]: I was trying to comment on that line and then my internet died. That seems like a fine solution, but I think the name to be checked could be
`const name = _class || phylum;`
and exclude kingdom. It's not a big deal, but if the closest guess we can make is "it's more likely bacteria than archaea," that's probably a very flimsy guess.

Comment (vincerubinetti) [reply src/data/worker.ts]: We're controlling this data, so hopefully one of those should always be defined? If we do that suggestion, what should be the fallback at the end, `""`? `"NA"` and skip?

**Review by rabdill — APPROVED** [review]
This looks fine to me! I'm not sure if you are looking for anything in particular in the review, but feel free to merge whenever you're ready.

**Pull request merged**